client/my-sites/plans/jetpack-plans/product-lightbox/payment-plan.tsx

```diff
@@ -13,11 +13,15 @@ type PaymentPlanProps = {
 	isMultiSiteIncompatible?: boolean;
 	siteId: number | null;
 	product: SelectorProduct;
+	hideTitle?: boolean;
+	showBorder?: boolean;
 };
 const PaymentPlan: React.FC< PaymentPlanProps > = ( {
 	isMultiSiteIncompatible,
 	siteId,
 	product,
+	hideTitle = false,
+	showBorder = false,
```

Comment:
Maybe we can describe the state here rather than the UI—something like `isActive`?